docs/resources/data-representation/constants.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: constants
+title: Constants
+tags: []
+description: Explore the importance of using Constants in FlutterFlow to define unchanging values throughout your application.
+sidebar_position: 5
+---
+
+# Constants
+Constants are used to define values that remain unchanged throughout the lifetime of an application. Using constants is a good practice for values that do not need to be recalculated or reassigned.
+
+Here are some real-world examples where using constants is beneficial:
+
+- **Educational Apps**: Constants for course-related data like the number of questions in a quiz or passing grade percentage.
+- **Fitness Apps**: Standard values like target heart rate zones or calorie burn rates based on specific exercises.
+- **Game Development**: Constants for game settings like default player health, speed, or levels. For example, `const int maxHealth = 100;`.
```

Comment:
Doesnt make sense to talk about code snippet for constants as it wil confuse non devs

Reply:
Ok, removed.